va/caa.go

```diff
@@ -113,35 +114,10 @@ func (va *ValidationAuthorityImpl) DoCAA(ctx context.Context, req *vapb.IsCAAVal
 			}
 			return remoteva.DoCAA(ctx, checkRequest)
 		}
-		var remoteProb *probs.ProblemDetails
-		summary, remoteProb = va.doRemoteOperation(ctx, op, req)
-		// If the remote result was a non-nil problem then fail the CAA check
-		if remoteProb != nil {
-			prob = remoteProb
-			va.log.Infof("CAA check failed due to remote failures: identifier=%v err=%s",
-				ident.Value, remoteProb)
-		}
+		summary, prob = va.doRemoteOperation(ctx, op, req)
 	}
 
-	if prob != nil {
-		// The ProblemDetails will be serialized through gRPC, which requires UTF-8.
-		// It will also later be serialized in JSON, which defaults to UTF-8. Make
-		// sure it is UTF-8 clean now.
-		prob = filterProblemDetails(prob)
-		return &vapb.IsCAAValidResponse{
-			Problem: &corepb.ProblemDetails{
-				ProblemType: string(prob.Type),
-				Detail:      replaceInvalidUTF8([]byte(prob.Detail)),
```

Comment:
We're losing a `replaceInvalidUTF8` call here. Is that intentional?

Maybe that call should be inside `ProblemDetailsToPB` anyhow, since it's in there because (IIRC) protobuf serialization is unhappy about invalid UTF-8.